packages/create-cli/package.json

```diff
@@ -40,19 +40,18 @@
   },
   "homepage": "https://github.com/checkly/checkly-cli#readme",
   "dependencies": {
+    "@esm2cjs/execa": "6.1.1-cjs.1",
     "@oclif/core": "2.8.11",
     "@oclif/plugin-help": "5.1.20",
     "@oclif/plugin-plugins": "2.3.0",
     "axios": "1.3.1",
     "chalk": "4.1.2",
     "debug": "4.3.4",
-    "execa": "5.1.1",
```

Comment:
I think `execa 5.1.1` should still be Common JS. According to the release log, they bumped the version in v6.0.0. Do we still need to use `@esm2cjs2/execa`?

https://github.com/sindresorhus/execa/releases

Reply:
@clample , You are right. But `execa@5.1.1` compiles but when it runs returns that it need ESM.

Reply:
The older versions of execa should work with commonjs. Let's not use a fork nobody knows

Reply:
Agree with the fork usage. I'll try `execa@5.1.1` again. I wasn't able to get it work: it compiles but when it runs it throw a 'ESM required' error. I'll share the issue here.